### DISCUSSION TITLE
Screen sharing type videos

### DISCUSSION THREAD

**Discussion by ismaelrumzan** [remotion-dev/remotion#932] [Q&A]
Hi,
I'm wondering if this is possible and what would be the best approach
I'd like to create code blocks or react component blocks that would be the video background and then highlight different aspects of the over time.
Basically, instead of recording screen captures of explaining different parts of a code block or how to use a specific rendered react component (that is used by a web platform). It would or would not have audio over

**Comment by JonnyBurger** [comment]
Hi Ismael!

Thanks for the question. Sounds like a good fit - I would recommend using some kind of react component that can syntax-highlight the code (had a good experience using react-prism-renderer in https://github.com/remotion-dev/trailer) and then put some overlays on top of it!

Also a good project that works with code highlights: https://github.com/pomber/record-talk-with-remotion

**Comment by tomByrer** [comment]
@ismaelrumzan  Liqvid might be a better fit, since it was built for code & math tutorials:
https://liqvidjs.org/

(Some talk @ a Remotion crossover, so not total poach.)
